supplier_app/tests/test_view.py

```diff
@@ -114,8 +114,8 @@ def setUp(self):
             user=self.user_with_eb_social
         )
         self.POST = taxpayer_creation_POST_factory(
-            self.file_mock,
-            self.bank_info
+            afip_file=self.file_mock,
```

Comment:
Should we rename this `afip_file` to something generic?

Reply:
mm, I think something generic could be confusing for others